pkg/operator/certrotationtimeupgradeablecontroller/certrotationtime_upgradeable.go

```diff
@@ -38,7 +38,7 @@ func NewCertRotationTimeUpgradeableController(
 	return factory.New().WithInformers(
 		operatorClient.Informer(),
 		configMapInformer.Informer(),
-	).WithSync(c.sync).ResyncEvery(time.Second).ToController("CertRotationTimeUpgradeableController", eventRecorder.WithComponentSuffix("certRotationTime-upgradeable"))
+	).WithSync(c.sync).ResyncEvery(1*time.Second).ToController("CertRotationTimeUpgradeableController", eventRecorder.WithComponentSuffix("certRotationTime-upgradeable"))
```

Comment:
seems way to tight

Reply:
It does seem tight. Although I though it was previously set to react to cert life cycle with 1s accuracy - I we want to test shortened rotation periods of say 10 or 5 minutes this needs to granular enough not to miss critical rotation.

What's the biggest lag you think we can afford? I was fine with the 1s as it doesn't result in external action or using live client afaik

Reply:
how is 30 sec or so to granular to catch something for rotation of 5 min?

Reply:
I was only saying it's a choice and it doesn't seem to do any live calls unless a change is desired. We may be fine with rotating at 3m instead of 2:30. If we claim it as tolerable lag, 30 seconds seem reasonable.

Reply:
Apologies, I was actually talking about another controller. This doesn't actually need a resync at all. It's purely based on object state and both informers are hooked.